be/src/http/action/compaction_action.cpp

```diff
@@ -61,7 +61,7 @@ CompactionAction::CompactionAction(CompactionActionType ctype, ExecEnv* exec_env
                                    TPrivilegeType::type ptype)
         : HttpHandlerWithAuth(exec_env, hier, ptype), _engine(engine), _type(ctype) {}
 
-/// check param and fetch tablet_id from req
+/// check param and fetch tablet_id & table_id from req
```

Comment:
You should modify the comment in cloud compaction action too

Reply:
done